src/components/Store.js

```diff
@@ -25,7 +29,7 @@ export const Store = () => {
       </header>
         <ToastContainer />
       <main>
-        {isProduct ? <ProductList products={PRODUCTS} addToCart={addToCart}/> : <Cart products={cartProducts}/>}
+        {isProduct ? <ProductList products={PRODUCTS} addToCart={addToCart}/> : <Cart cartItems={cartProducts} removeFromCart={removeFromCart}/>}
```

Comment:
NIT: We could format it better in order to make it more readable.

```suggestion
        {isProduct
        ? <ProductList products={PRODUCTS} addToCart={addToCart}/>
        : <Cart cartItems={cartProducts} removeFromCart={removeFromCart}/>}
```